plugins/populator/frontend/public/javascripts/countly.models.js

```diff
@@ -436,13 +481,11 @@
         };
 
         this.getError = function() {
-            var errors = [];
+            var errors = "";
```

Comment:
Why is this variable changed into a string?
It's no longer used for web platform, but it is still used in other platforms as an array, so it could be confusing if the type is changed.